_config.yml

```diff
@@ -9,7 +9,7 @@ url: https://theforeman.org
 baseurl: "" #keep at "" for the Atom feed
 
 # latest *stable* version
-foreman_version: '1.24'
+foreman_version: '2.0'
```

Comment:
This won't get updated until Foreman 2.0 is GA and someone on the Foreman side will update it